Enough/src/EnoughCoffee.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner;
+
+public class EnoughCoffee {
+    public static void main(String[] args) {
+        int water = 200;
+        int milk = 50;
+        int coffeeBeans = 15;
+        System.out.println("Hi! I will tell you how many cups of coffee can I make, if you answer these questions");
+        System.out.println("how much water do I have?");
+        Scanner enter = new Scanner(System.in);
+        int myWater = enter.nextInt();
+        System.out.println("how much milk do I have?");
+        int myMilk = enter.nextInt();
+        System.out.println("how many grams of coffee beans do I have?");
+        int myCoffeeBeans = enter.nextInt();
+        System.out.println("How many cups of coffee do you want?");
+        int cups = enter.nextInt();
+        int enoughWater = myWater / water;
+        int enoughMilk = myMilk / milk;
+        int enoughCoffeeBeans = myCoffeeBeans / coffeeBeans;
+        int smallest;
+        if (enoughWater <= enoughCoffeeBeans && enoughWater <= enoughMilk) {
```

Comment:
Can you separate code for different small functions ?

Reply:
you mean to write several if-else blocks?